components/resolved_ts/src/endpoint.rs

```diff
@@ -806,19 +815,21 @@ where
                 }
             }
         }
+        let now = TimeStamp::physical_now();
+        RTS_MIN_SAFE_TS.set(oldest_safe_ts as i64);
+        RTS_MIN_SAFE_TS_REGION.set(oldest_safe_ts_region as i64);
+        RTS_MIN_SAFE_TS_GAP
+            .set(now.saturating_sub(TimeStamp::from(oldest_safe_ts).physical()) as i64);
```

Comment:
Should we also add a "slow log" here if the gap or the slowness exceeds a specific interval so we may search for such information in logs too?